pkg/core/hostid/aws.go

```diff
@@ -21,7 +21,7 @@ func AWSUniqueID(cloudMetadataTimeout timeutil.Duration) string {
 	if err != nil {
 		log.WithFields(log.Fields{
 			"error": err,
-		}).Debug("Failed to get AWS instance-identity")
+		}).Info("Failed to get AWS instance-identity - It's possible the agent is not running on AWS")
```

Comment:
```suggestion
		}).Info("No AWS metadata server detected, assuming not on EC2.")
```

That is a bit more explicit and has less potential for confusion.  It's overwhelming likely that they aren't on AWS, not just "possible".

Reply:
done